pal/samples/esp8266/telemetry_sample.c.ino

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <AzureIoTUtility.h>
+
+// CAVEAT: This sample is to demonstrate azure IoT client concepts only and is not a guide design principles or style
+// Checking of return codes and error values shall be omitted for brevity.  Please practice sound engineering practices
+// when writing production code.
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "iothub.h"
+#include "iothub_client.h"
+#include "iothub_device_client_ll.h"
+#include "iothub_client_options.h"
+#include "iothub_message.h"
+#include "azure_c_shared_utility/crt_abstractions.h"
+#include "azure_c_shared_utility/shared_util_options.h"
+#include "azure_c_shared_utility/tlsio.h"
+#include "azure_c_shared_utility/xlogging.h"
+
+#define SET_TRUSTED_CERT_IN_SAMPLES
+#ifdef SET_TRUSTED_CERT_IN_SAMPLES
```

Comment:
are the certs always supposed to be in here? if they are we might want to think about just including `certs.h` instead of this. seems kind of odd to set and then check if set one after the other

Reply:
or maybe in some config? i'm not exactly sure how arduino compilation works

---
In reply to: [342691497](https://github.com/Azure/azure-iot-pal-arduino/pull/7#discussion_r342691497) [](ancestors = 342691497)

Reply:
We probably do just always want certs, I'll get rid of the define.

Reply:
Done.